src/main/java/io/anserini/collection/TwitterCollection.java

```diff
@@ -0,0 +1,96 @@
+package io.anserini.collection;
+
+import io.anserini.document.TwitterDocument;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.zip.GZIPInputStream;
+
+/**
+ * Class representing an instance of a TREC Microblog collection.
```

Comment:
TREC Microblog or Twitter? I know they're essentially the same, but when the class is called `TwitterCollection` it's probably best that the comment matches that.